dist/tools/pktbuf-stats/pktbuf-stats.py

```diff
@@ -0,0 +1,524 @@
+#! /usr/bin/env python3
```

Comment:
The tools looks good, and having it is a plus to understand those nice `pktbuf` dumps :-) I'd like to have a bit more of documentation on the script (e.g. doctrings) and maybe also having a list of the types which can be decoded.

Reply:
Sure, can do.

Reply:
ping @miri64 :)

Reply:
I put it on my todo list ^^"

Reply:
Done.